### DISCUSSION TITLE
Zksyc contract deployment failure

### DISCUSSION THREAD

**Discussion by JusticeSol** [Cyfrin/foundry-full-course-cu#2326] [General]
Hello guys, I'm stocked at deploying my contract on zksync sepolia testnet, all steps are correct but it keeps displaying "Deploying contract SimpleStorage" for a long while now.

**Comment by EngrPips** [comment]
Hey @HybornCV, have You figured this out already?

**Comment by Iammrgemini** [comment]
facing this same problem, any solution ? i am getting this Deploying contract SimpleStorage and nothing more

**Reply by EngrPips** [reply]
If you have followed all the necessary steps and you are still running into issues, leave it and proceed. Sepolia testnet is sufficient for everything you will learn.

**Reply by JusticeSol** [reply]
@Iammrgemini Go ahead and deploy the contract directly, do not check the "audit/verify" contract box.
